### PR TITLE
Add a util for explorer functionality + unit tests

### PR DESCRIPTION
**Pull request by janedegtiareva**
near send will now print a link to explorer url at the end of the output, if there is a known explorer url and also if transaction id is known. 

This is needed for https://github.com/near/near-shell/issues/410

### CODE REVIEW

**Comment by mikedotexe** [comment]
This is great! I think the full log is more verbose than we want to be. I think the transaction hash is all that we really want if I'm understanding this ticket correctly.

Screenshot 2020-07-09 14 29 44

Screenshot 2020-07-09 14 27 30

Link worked great to me :)

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "test": "./test/index.sh",
+    "test": "jest && ./test/index.sh",
```
Comment (mikedotexe) [inline package.json:10]: this is causing tests to fail

**Comment by mikedotexe** [comment]
Looks like there's linting issues in three files, too

File: package.json (hunk 1 above)
Comment (janedegtiareva) [reply package.json:10]: It's a different issue (1 unfixed reference after refactoring). Addressed.

**Comment by janedegtiareva** [comment]
> Looks like there's linting issues in three files, too

I already fixed linter issues

**Comment by janedegtiareva** [comment]
re full response too verbose: I agree, and it also is not very user friendly. I was going to address that in a follow up PR so that the actual explorer work lands with a smaller surface area.

**Comment by mikedotexe** [comment]
> re full response too verbose: I agree, and it also is not very user friendly. I was going to address that in a follow up PR so that the actual explorer work lands with a smaller surface area.

I think it's got everything we need, just call `getTxnId` so it says
Transaction ID: 234kljsdklfjk3l43
View this transaction:
https://explorer.…

**Pull request merged**